f5/bigip/tm/sys/test/functional/test_management_ip.py

```diff
@@ -0,0 +1,109 @@
+# Copyright 2015-2106 F5 Networks Inc.
```

Comment:
when did we start adding the shebang to files? I haven't seen that in the other modules